packages/react/src/components/builder-component.component.tsx

```diff
@@ -1113,9 +1113,12 @@ export class BuilderComponent extends React.Component<
                           <div
                             data-builder-component={this.name}
                             data-builder-content-id={fullData.id}
-                            data-builder-variation-id={
-                              fullData.testVariationId || fullData.variationId || fullData.id
-                            }
+                            {...(this.isPreviewing
+                              ? {
+                                  'data-builder-variation-id':
```

Comment:
Are we sure we do not rely on this attribute in any scripts? Also, do we have any documentation that references this attribute anywhere, in case a customer is relying on it being there?

Reply:
it's only used by editor